misc-utils/lsfd-unkn.c

```diff
@@ -663,12 +659,12 @@ static char *anon_timerfd_get_name(struct unkn *unkn)
 	clockid_name = anon_timerfd_decode_clockid(data->clockid);
 
 	exp = &data->itimerspec.it_value;
-	if (!is_zero_timespec(exp))
+	if (!is_timespecset(exp))
```

Comment:
The names `is_zero_timespec` and `is_timespecset` are negations of each other.
So the `!` should be removed here and the logic in the function changed to `t->tv_sec || t->tv_nsec`.

Reply:
Thank you. I'm ashamed.

Reply:
Don't be! It's such an easy thing to miss.